content/en/docs/v3.5/install.md

```diff
@@ -86,14 +86,122 @@ $ brew update
 $ brew install etcd
 ```
 
-3. Verify install 
+3. Verify install
 ```sh
 $ etcd --version
 ```
 
 ## Linux
 
-TBD
+Although it is possible to install etcd through major distribution's official repositories and package managers, the versions published are significantly outdated. So, installing this way is strongly discouraged.
+
+The following package managers are community maintained and have the most up-to-date version:
+
+### Homebrew on Linux
+
+[Homebrew can run on Linux], and can provide recent software versions.
+
+* Prerequisites
```

Comment:
Suggest running markdownlint to conform with best practices.

> MD004/ul-style: Unordered list style [Expected: dash; Actual: asterisk]markdownlint[MD004](https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md004.md)

Reply:
Would it make sense to add a GitHub action to lint the markdown files?

Reply:
Yes - But probably in a permissive mode initially as it will open a can of worms lol.

Reply:
Maybe we can start incrementally by linting the diffs. I'll put together a POC if it makes sense.

Reply:
Sorry missed this ping earlier @ivanvc - Agree let's open a separate issue for markdownlint improvements.

Reply:
I'll open an issue to track this.